src/System.Private.Xml/tests/Writers/XmlWriterApi/TCFullEndElement.cs

```diff
@@ -640,8 +640,8 @@ public override void AddChildren()
                 // for function CData_13
                 {
                     this.AddChild(new CVariation(CData_13) { Attribute = new Variation("WriteCData with 0x0A with NewLineHandling.Entitize") { Params = new object[] { 10, NewLineHandling.Entitize, "<r><![CDATA[\n]]></r>" }, id = 18, Pri = 1 } });
-                    this.AddChild(new CVariation(CData_13) { Attribute = new Variation("WriteCData with 0x0D with NewLineHandling.Replace") { Params = new object[] { 13, NewLineHandling.Replace, "<r><![CDATA[\r\n]]></r>" }, id = 13, Pri = 1 } });
```

Comment:
Here I actually only fixed the failing outerloop tests. I'm working on cleaning up all Xml tests (https://github.com/dotnet/corefx/issues/14071) right now and have that change in there.